zerver/lib/import_realm.py

```diff
@@ -455,6 +464,66 @@ def fix_message_rendered_content(
             )
 
 
+def remap_near_link_recipient_encoding(fragments: list[str]) -> None:
+    section, recipient_encoding = fragments[1], fragments[2]
+    old_recipient_id, recipient_name = recipient_encoding.split("-", maxsplit=1)
```

Comment:
this `split` is called with `maxsplit=1` if the channel name contains spaces, those will be replaced with "-"